h/static/scripts/controllers/lozenge-controller.js

```diff
@@ -18,9 +19,21 @@ class LozengeController extends Controller {
   constructor(containerEl, options) {
     super(containerEl, options);
     var lozengeEl = document.createElement('div');
+    var lozengeMarkup = options.content;
+
+    if(searchTextParser.hasKnownNamedQueryTerm(options.content)) {
```

Comment:
Style nit - there should be a space after `if`. Our lint config needs updating to check that.